config/interceptors/core-interceptors.yaml

```diff
@@ -16,6 +16,9 @@ apiVersion: triggers.tekton.dev/v1alpha1
 kind: ClusterInterceptor
 metadata:
   name: cel
+  labels:
+    server/type: https
```

Comment:
If we need this, then I think we should make it part of the spec and not a label

Reply:
Sure will do that

Reply:
I think having `TLS` only isn't good. There will always be user who don't want added benefit of TLS for their microservice running in private servers. We should keep `HTTP` also. So having a field is good.

Reply:
As discussed in Triggers WG call
As  part of this PR we are keeping label itself if require may be we can add as a field as part of another PR